tests/unit_tests/test_game_rating.py

```diff
@@ -189,7 +201,41 @@ async def test_on_game_end_global_ratings(custom_game, players):
     assert results.outcomes[players.joining.id] is GameOutcome.DEFEAT
 
 
-async def test_on_game_end_ladder_ratings_(ladder_game, players):
+async def test_on_game_end_global_ratings_published(custom_game, players):
+    rating_service = custom_game.game_service._rating_service
+    custom_game.state = GameState.LOBBY
+    add_connected_players(custom_game, [players.hosting, players.joining])
+    custom_game.set_player_option(players.hosting.id, "Team", 2)
+    custom_game.set_player_option(players.joining.id, "Team", 3)
+
+    await custom_game.launch()
+    await custom_game.add_result(players.hosting.id, 0, "victory", 1)
+    await custom_game.add_result(players.joining.id, 1, "defeat", 0)
+
+    custom_game.enforce_rating = True
+    await custom_game.on_game_end()
+    await rating_service._join_rating_queue()
+
+    results = get_published_results_by_player_id(
+        custom_game.game_service._rating_service
+    )
+    assert players.hosting.id in results
+    assert players.joining.id in results
+
+    host_result = results[players.hosting.id]
+    assert host_result["rating_type"] is RatingType.GLOBAL
```

Comment:
I don't think you want `is` here as `RatingType.GLOBAL` is just a string constant (not an enum)

Reply:
Ah yes, I forgot that that changed a while ago...